src/Application/Components/Store/Store.jsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import StoreItem from "./StoreItem";
+
+const Store = React.createClass({
+  propTypes: {
+    assets: React.PropTypes.array.isRequired,
+    createOnClick: React.PropTypes.func.isRequired,
+  },
+
+  componentDidMount() {
+    this.props.assets[0].unlock();
+    this.forceUpdate();
+  },
+
+  render() {
+    let unlockedAssets = [];
+    for (const asset of this.props.assets) {
+      if (asset.unlocked) {
+        unlockedAssets.push(asset);
+      }
+    }
+
+    const buyFxn = this.props.createOnClick;
+
+    return (
+        <div id="store-zone">
+          <h3>Broker</h3>
+          <div id="item-menu">
+            {unlockedAssets.map(function (asset) {
+              return <StoreItem key={asset.name} name={asset.name} price={asset.price} owned={asset.owned} onClick={buyFxn(asset)} />;
```

Comment:
Agreed, how do you handle something like this?
The button needs the info like Name, Price, #Owned, and onclick
I can avoid the key somehow by not nesting this? These are all on the Asset object, but I thought that was bad practice to send more than you need, and an object.